packages/model-viewer/src/test/features/scene-graph/material-spec.ts

```diff
@@ -34,18 +34,23 @@ suite('scene-graph/material', () => {
     let element: ModelViewerElement;
     let texture: Texture|null;
 
-    const init = async () => {
+    setup(async () => {
       element = new ModelViewerElement();
       element.src = HELMET_GLB_PATH;
       document.body.insertBefore(element, document.body.firstChild);
       await waitForEvent(element, 'load');
 
       texture = await element.createTexture(REPLACEMENT_TEXTURE_PATH);
-    };
 
-    test('Set a new base map', async () => {
-      await init();
+      await timePasses();
```

Comment:
Were the tests not passing without this extra await? Any idea why this would be needed?

Reply:
I pulled this over from example code and never removed it, it seems to run fine without it.